rust_dev_preview/tls/README.md

```diff
@@ -0,0 +1,65 @@
+# AWS SDK for Rust code examples for TLS
+
+## Purpose
+
+These examples demonstrate how to adjust TLS settings to custom values. This can be useful to enforce
+custom levels of security. For instance, if your company requires a minimum version of 1.3 for outgoing calls.
```

Comment:
Update second and third sentences: 
This can help you enforce custom levels of security. For example, you might require a minimum version of TLS 1.3 for outgoing calls.